make error = (msg) does
    print(msg)
end
error("fail")
